actions/pause

```diff
@@ -7,8 +7,8 @@ export PATH=$PATH:/snap/bin
 DELETE_LOCAL_DATA=$(action-get delete-local-data)
 FORCE=$(action-get force)
 
-# placeholder for additional flags to the command
-export EXTRA_FLAGS=""
+# Needed to drain CDK nodes.
+export EXTRA_FLAGS="--ignore-daemonsets --delete-local-data"
```

Comment:
why are we ignoring daemonsets?

Reply:
Because of `nginx-ingress-controller-kubernetes-worker`.

Reply:
I know it is a daemonset, but why are we not killing them off when we cordon? Why add this flag? What bug or issue are we trying to get around?

Reply:
Ah, because otherwise the `drain` command fails silently.  I raised this in a PL last week (for context).